converter/converter.py

```diff
@@ -1,8 +1,8 @@
 from converter.helper import add_basic_keywords, add_data_keywords
 
 class Converter():
-    def __init__(self, data_keywords_mapper, kwarg_to_schema_key_mapper, required_keywords):
-        self.required_keywords = ["provider_address", "program_name", "program_url"] + required_keywords
+    def __init__(self, data_keywords_mapper, kwarg_to_schema_key_mapper, additional_required_keywords=[]):
+        self.required_keywords = ["provider_address", "program_name", "program_description", "program_url"] + additional_required_keywords
```

Comment:
To prevent an error from occurring if users don't pass additional_required_keywords as a list, could we add a check like
```python
if type(additional_required_keywords) is str:
    additional_required_keywords = [additional_required_keywords]
```

I don't think we're exporting the converter class so maybe it doesn't matter. Up to you.

Reply:
I think this only applies when someone extends the library to support another format.

Reply:
I think this check makes sense, and I like giving the user alternative ways to pass `required_keywords`. I added it over here: https://github.com/brighthive/google-pathways-converter/pull/12/commits/ad66664ea8738cb5f7aed5dc377de3124b2d3584

N.b., I also undid a previous change, from an earlier commit in this PR:
```
self.required_keywords = ["provider_address", "program_name", "program_description", "program_url"] + additional_required_keywords
```
I can imagine having program types beyond WorkBased and EducationalOccupational, which do not require these fields. Sooooo.......extensibility!